frontend/src/theme/darkTheme/themeColors.ts

```diff
@@ -40,4 +40,12 @@ export default {
   blue: colors.blue80,
   green: colors.green80,
   yellow: colors.yellow80,
+
+  // colors used to colour markdown with remark-directive (darkTheme)
+  mdRed: "#FF4B4B",
```

Comment:
nit: Just one small consideration to make here - maybe ask the design team: This will only support dark colors if the actual `dark` theme is selected. A custom theme with dark background color will still use light colors.

Reply:
Is there an easy fix/workaround for this? ex. variable which indicates it's custom theme with dark background color?